src/main/java/uk/gov/hmcts/ccd/domain/types/CustomTypes.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.ccd.domain.types;
+
+/**
+ * Custom types IDs
+ */
+public enum CustomTypes {
+
+    CASE_LINK_TEXT_CASE_REFERENCE("TextCaseReference"),
+    ORG_POLICY_CASE_ASSIGNED_ROLE("OrgPolicyCaseAssignedRole");
```

Comment:
I forgot to remove line 9 can you remove it please

Reply:
DONE